examples/tests/crawl_and_generate_SBOM.py

```diff
@@ -0,0 +1,94 @@
+import json
+import os
+import re
+import subprocess
+import tempfile
+from urllib.parse import urlparse
+
+import requests
+from git import Repo
+
+SBOM_OUTPUT = "./SBOM/"
```

Comment:
It would be beneficial to allow the user to specify the `SBOM_OUTPUT` directory. This could be done by checking for an environment variable and fallback to "./SBOM".